data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/CloudEventRequestToRecordMapper.java

```diff
@@ -52,19 +50,18 @@ public CloudEventRequestToRecordMapper(final Vertx vertx) {
         }
 
         if (logger.isDebugEnabled()) {
-          final var span = TracingSpan.getCurrent(vertx);
+          final var span = Span.fromContextOrNull(Context.current());
```

Comment:
What current means here? Shouldn't we pull the span from the Vertx context?

Reply:
Can we  use a utility to get the current span? The idea being isolating OTEL code in one module.

Reply:
> What current means here? Shouldn't we pull the span from the Vertx context?

Nope, I've integrated the vert.x context with the otel context apis https://github.com/eclipse-vertx/vertx-tracing/blob/master/vertx-opentelemetry/src/main/java/io/vertx/tracing/opentelemetry/VertxContextStorageProvider.java, so to retrieve spans we don't need to access the vert.x context manually.

> Can we use a utility to get the current span? The idea being isolating OTEL code in one module.

TBH I think we don't need to hide the otel apis, they are straightforward and using them directly sounds reasonable to me...